packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/selectors.ts

```diff
@@ -126,10 +125,20 @@ export const getData = (state: RootState, ownProps: OwnProps) => {
       coinSelectorMap(state, coin),
       selectors.core.settings.getCurrency,
       selectors.components.recurringBuy.getRegisteredListByCoin(coin),
-      selectors.core.walletOptions.getFeatureFlagRecurringBuys
+      selectors.core.walletOptions.getFeatureFlagRecurringBuys,
+      () => selectors.components.interest.getInterestEligible(state)
     ],
-    (invitationsR, userSearch, pagesR, currencyR, recurringBuys, isRecurringBuyR) => {
+    (
+      invitationsR,
+      userSearch,
+      pagesR,
+      currencyR,
+      recurringBuys,
+      isRecurringBuyR,
+      interestEligibleResult
+    ) => {
       const empty = (page) => isEmpty(page.data)
+      const interestEligible = propOr(false, 'data', interestEligibleResult)
```

Comment:
you need to map over the Remote using one of its instance methods or use a custom to get the value. never access `data` directly

Reply:
I think this is fixed now. Please review @schnogz when you can.